azure-pipelines.yml

```diff
@@ -656,13 +656,17 @@ jobs:
       set -ev
       # Force relink python@3.xx in Homebrew to resolve the conflict with pre-installed python 3.xx on macOS-12 image
       # See: https://github.com/Azure/azure-cli/issues/29054
-      python_version=3.12
-      brew unlink python@$python_version && brew link --overwrite python@$python_version
+      brew unlink python@$3.12 && brew link --overwrite python@$3.12
+      brew unlink python@3.13 && brew link --overwrite python@3.13
```

Comment:
`python@3.13` is a dependency of azure-cli:
`==> Installing dependencies for azure-cli: libgit2, sqlite, python@3.13, z3, llvm, rust and python@3.12
`
3.13.5 just released, and brew tries to update it. We need to manually overwrite it too. 

Ref: https://pythoninsider.blogspot.com/2025/06/python-3135-is-now-available.html

Reply:
See https://stackoverflow.com/questions/25794905/why-does-set-e-true-false-true-not-exit